.generation/generate.py

```diff
@@ -208,6 +209,11 @@ def generate_typescript_code(*, npm_name: str, npm_version: str, repository_url:
         ],
         check=True,
     )
+    with open(Path("typescript") / ".gitignore", 'w', encoding='utf-8') as f:
```

Comment:
why did you add this?

Reply:
Otherwise, the generator would put "dist" there. At the moment the `dist` folder exists in the `main` branch. When ignoring that folder, it won't get updated.